crates/language-tests/tests/language/statements/define/field/flexible_array_any.surql

```diff
@@ -23,7 +23,7 @@ value = "[{ custom: [{ key: 'sometext' }], id: user:three }]"
 
 DEFINE TABLE user SCHEMAFULL;
 DEFINE FIELD custom ON user TYPE option<array>;
-DEFINE FIELD OVERWRITE custom.* ON user FLEXIBLE TYPE any;
```

Comment:
hmm you can argue that flexible should be allowed with the `any` type since it can technically store an object

Reply:
It's redundant, `any` allows anything. There is no functional difference between `TYPE any` and `TYPE any FLEXIBLE`.